### ISSUE TITLE
PyVista may be using QWidget instead of QGLWidget for VTK 8.2

### ISSUE THREAD

**Issue by math-artist** [pyvista/pyvista#473]
I encountered a problem and found out that QVTKRenderWindowInteractor is settting QWidget even after the line 
`vtk.qt.QVTKRWIBase = 'QGLWidget'` is executed.

It turns out that QVTKRenderWindowInteractor uses the vtkmodules namespace, which is different than vtk.

Replace the line `vtk.qt.QVTKRWIBase = 'QGLWidget'` for
`vtkmodules.qt.QVTKRWIBase = 'QGLWidget'` to solve the problem with 8.2. You also need 
`import vtkmodules.qt` instead of `import vtk.qt`

**Comment by GuillaumeFavelier** [comment]
Wow! How did you find that `QVTKRenderWindowInteractor` sets `QWidget` or uses `vtkmodules`?

**Comment by math-artist** [comment]
I compile my VTK project with Nuitka, and the get-hints feature didn't catch the GPLWidget import properly. My app crashed at runtime.

**Comment by GuillaumeFavelier** [comment]
Nice catch. By looking into the [VTK code](https://github.com/Kitware/VTK/blob/master/Wrapping/Python/vtkmodules/qt/QVTKRenderWindowInteractor.py#L50-L59), I found this:
```
# Check whether a specific PyQt implementation was chosen
try:
    import vtkmodules.qt
    PyQtImpl = vtkmodules.qt.PyQtImpl
except ImportError:
    pass

# Check whether a specific QVTKRenderWindowInteractor base
# class was chosen, can be set to "QGLWidget"
QVTKRWIBase = "QWidget"
```

**Comment by math-artist** [comment]
exactly.
The line `QVTKRWIBase = vtkmodules.qt.QVTKRWIBase` uses vtkmodules and the line in qt_plotting.py uses vtk... The value is stored but never used.

**Comment by GuillaumeFavelier** [comment]
If I understood correctly, the fix should be fairly short. Would you mind opening a PR @math-artist ?

**Comment by banesullivan** [comment]
@GuillaumeFavelier, do you know what needs to be done for this? Could you patch a quick fix for this as `0.23.0` is coming soon

**Comment by GuillaumeFavelier** [comment]
@math-artist I opened #498 to fix this, could you try it out to see if it fixes your app crash?
